[April 27, 2023 9AM PST] dbt Cloud GA for Starburst/Trino adapter support

nghi-ly opened the pull request:
## What are you changing in this pull request and why?
To support the GA of the Starburst/Trino adapter in dbt Cloud, remove the public preview calllout and update prerequisites in the quickstart: 

https://docs.getdbt.com/docs/quickstarts/dbt-cloud/starburst-galaxy

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] **DO NOT MERGE** this PR until launch day

nghi-ly committed:
Remove public preview calllout

mirnawong1 committed:
starburst-trino-ga release note

release note to accompany apr 27th ga for starburst/trino ga. cc @Fleid

nghi-ly committed:
Merge branch 'current' into ly-docs-starburst-ga

nghi-ly committed:
Merge branch 'current' into ly-docs-starburst-ga

mirnawong1 committed:
Merge branch 'current' into ly-docs-starburst-ga

runleonarun commented on website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md:
```diff
@@ -17,13 +17,8 @@ In this quickstart guide, you'll learn how to use dbt Cloud with [Starburst Gala
-
 ## Prerequisites 
-- You have a [dbt Cloud account](https://www.getdbt.com/signup/).
+- You have [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment in [dbt Cloud](https://www.getdbt.com/signup/). For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy).
```
```suggestion
- You have a [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment in [dbt Cloud](https://www.getdbt.com/signup/). For more information, refer to [Tenancy](/docs/cloud/about-cloud/tenancy).
```

runleonarun reviewed (dismissed):
Approved for GA on Thursday.

runleonarun committed:
Update website/docs/docs/quickstarts/dbt-cloud/starburst-galaxy-qs.md

runleonarun committed:
Update website/docs/docs/dbt-versions/release-notes/09-April-2023/starburst-trino-ga.md

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-12

mirnawong1 committed:
Merge branch 'current' into ly-docs-starburst-ga

mirnawong1 committed:
update trino setup and add to cloud adapter about page

per [slack thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1682422591945219?thread_ts=1681986411.262729&cid=C02NCQ9483C) and confirmed by PM

mirnawong1 committed:
Merge branch 'current' into ly-docs-starburst-ga

mirnawong1 committed:
Update about-connections.md

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-22

mirnawong1 committed:
Merge branch 'current' into ly-docs-starburst-ga

mirnawong1 committed:
Merge branch 'current' into mirnawong1-patch-12

mirnawong1 committed:
Merge branch 'ly-docs-starburst-ga' into mirnawong1-patch-22

mirnawong1 committed:
Merge branch 'ly-docs-starburst-ga' into mirnawong1-patch-12

mirnawong1 committed:
update trino setup and add to cloud adapter about page (#3270)

small changes to starburst trino, this should be merged thurs pst (apr
27th) alongside pr #3220 GA launch

per [slack
thread](https://dbt-labs.slack.com/archives/C02NCQ9483C/p1682422591945219?thread_ts=1681986411.262729&cid=C02NCQ9483C)
and confirmed by PM.

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs,
see the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
- [ ] MERGE APR 27th

mirnawong1 committed:
Merge branch 'ly-docs-starburst-ga' into mirnawong1-patch-12

mirnawong1 committed:
starburst-trino-ga release note - to merge apr 27th (#3231)

release note to accompany apr 27th ga for starburst/trino ga. to merge
apr 27th

cc @Fleid

## What are you changing in this pull request and why?
<!---
Describe your changes and why you're making them. If linked to an open
issue or a pull request on dbt Core, then link to them here! 

To learn more about the writing conventions used in the dbt Labs docs,
see the [Content style
guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md).
-->

## Checklist
- [ ] MERGE APR 27th

mirnawong1 committed:
Merge branch 'current' into ly-docs-starburst-ga

runleonarun committed:
Update about-connections.md

runleonarun committed:
Merge branch 'current' into ly-docs-starburst-ga

runleonarun reviewed (approved):
This looks ready to 🚢! Approving for tomorrow's GA

mirnawong1 committed:
Merge branch 'current' into ly-docs-starburst-ga

runleonarun committed:
Merge branch 'current' into ly-docs-starburst-ga
